test/test_rospy/test/unit/test_rospy_core.py

```diff
@@ -116,6 +116,7 @@ def test_loggers(self):
             base, ext = os.path.splitext(this_file)
             if ext == '.pyc':
                 this_file = base + '.py'
+            this_file = re.escape(this_file)
```

Comment:
Please apply the escaping where it is being used - in this case directly in line 182.